src/builder_ext.rs

```diff
@@ -21,6 +21,8 @@ use crate::qdrant::{
     UpsertPointsBuilder, Value, VectorParamsBuilder, VectorsSelector, WithLookupBuilder,
 };
 
+//use crate::builders::CreateCollectionBuilder;
```

Comment:
Does this have any specific purpose?

Reply:
thanks for catching this, this is not required, initially i was testing it using direct imports from builders instead of export it from qdrant.rs so this I just missed deleting it. I will remove it in the next push.

Reply:
done https://github.com/qdrant/rust-client/pull/199/commits/779098315d2f8c5c8044f56f30954c569e0e969b